.circleci/config.yml

```diff
@@ -591,54 +614,24 @@ jobs:
   ### e2e/integration test jobs ###
   #################################
 
-  e2e:
-    machine:
-      image: ubuntu-2004:202010-01
-    resource_class: large
-    environment:
-      GOCACHE: /tmp/go-cache
-    working_directory: /home/circleci/go/src/github.com/influxdata/influxdb
+  e2e-monitor-ci:
+    docker:
+      - image: cimg/base:2021.04
```

Comment:
imo an LTS image would be preferable, e.g. `cimg/stable-20.04`, unless there's something specific in the 2021.04 image you need.